ethereum-operations/src/subagents/node-operators-registry/constants.ts

```diff
@@ -25,17 +43,26 @@ export const NODE_OPERATORS_REGISTRY_EVENTS_OF_NOTICE = [
     address: NODE_OPERATORS_REGISTRY_ADDRESS,
     event: "event NodeOperatorActiveSet(uint256 indexed id, bool active)",
     alertId: "NODE-OPERATOR-ACTIVE-SET",
-    name: "ℹ NO Registry: Node operator active set",
+    name: "ℹ️ NO Registry: Node operator active set",
     description: (args: any) =>
       `Node operator ${args.id} active status set to ${args.active}`,
     severity: FindingSeverity.Info,
   },
+  {
+    address: NODE_OPERATORS_REGISTRY_ADDRESS,
+    event: "event NodeOperatorNameSet(uint256 indexed id, string name)",
```

Comment:
```suggestion
    event: "event NodeOperatorNameSet(uint256 indexed nodeOperatorId, string name)",
```

Double-check, please.